client/my-sites/stats/geochart/index.jsx

```diff
@@ -18,17 +17,51 @@ module.exports = React.createClass( {
 	visualization: null,
 
 	propTypes: {
-		data: React.PropTypes.array.isRequired,
-		dataList: React.PropTypes.object.isRequired
+		data: PropTypes.array.isRequired,
+		dataList: PropTypes.object.isRequired
 	},
 
-	recordEvent: function() {
+	componentDidMount() {
+		const domNode = this.refs.chart;
+		const script = document.createElement( 'script' );
+
+		script.src = 'https://www.google.com/jsapi';
+
+		if ( ! window.google ) {
```

Comment:
Would it make sense to create the 'script' element after detecting `window.google`?